themes/digital.gov/src/scss/new/_topics-button.scss

```diff
@@ -0,0 +1,101 @@
+@use "uswds-core" as *;
+
+.topics-button {
+  margin-bottom: units("105");
+
+  &__link {
+    @include u-radius("sm");
+    background: white;
+    border: 1px color("primary-vivid") solid;
+    color: color("primary-vivid");
+    display: block;
+    font-family: family("sans");
+    font-size: font-size("sans", "sm");
+    font-weight: font-weight("medium");
+    overflow: auto;
+    text-decoration: none;
+    transition: background-color 0.5s, color 0.5s;
+    white-space: nowrap;
+
+    @include at-media("tablet") {
+      overflow: auto;
+      padding-left: 6px;
+      padding-right: 6px;
+    }
+  }
+
+  &__arrow {
+    @include u-float("right");
+    color: color("gray-1");
+    display: none;
+    margin-right: 10px;
+    transition: margin 0.4s;
+
+    @include at-media("desktop") {
```

Comment:
I noticed at certain viewport width the button arrow styles break out of layout, this happens on the live site too.

I changed breakpoint from `tablet-lg` to `desktop` for the `topics-button__arrow` to remove this visual layout glitch.

<img width="1498" alt="Screenshot 2024-03-20 at 1 50 11 PM" src="https://github.com/GSA/digitalgov.gov/assets/104778659/8c4e895d-93a8-4e5d-9ff8-b91eb42feb70">